tensorboard/main.py

```diff
@@ -40,7 +40,7 @@ def run_main():
 
     tensorboard = program.TensorBoard(
         plugins=default.get_plugins(),
-        subcommands=[uploader_subcommand.UploaderSubcommand()],
+        subcommands=[legacy_tbdev_subcommand.LegacyTbdevSubcommand()],
```

Comment:
Maybe it's not worthwhile but it seems like you could at least delete uploader_subcommand.py.